subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http/HttpServerFixture.groovy

```diff
@@ -48,6 +49,7 @@ trait HttpServerFixture {
     private AuthScheme authenticationScheme = AuthScheme.BASIC
     private boolean logRequests = true
     private final Set<String> authenticationAttempts = Sets.newLinkedHashSet()
+    private final Map<String, String> allHeaders = Maps.newHashMap()
```

Comment:
Shouldn't this be a `List<Map<String, String>>` so one can distinguish individual requests?

Reply:
There is only one place where this is used, and the List will contain only one element then. But I'll update it anyway, its cleaner then.